collectd/files/write_graphite.conf

```diff
@@ -10,16 +10,18 @@
 LoadPlugin write_graphite
 
 <Plugin write_graphite>
- <Carbon>
-   Host "{{ collectd_settings.plugins.write_graphite.host }}"
-   Port "{{ collectd_settings.plugins.write_graphite.port }}"
-   Prefix "{{ collectd_settings.plugins.write_graphite.prefix }}."
-   Postfix "{{ collectd_settings.plugins.write_graphite.postfix }}"
-   Protocol "{{ collectd_settings.plugins.write_graphite.protocol }}"
-   EscapeCharacter "{{ collectd_settings.plugins.write_graphite.escapecharacter }}"
-   LogSendErrors {{ collectd_settings.plugins.write_graphite.logsenderrors|lower }}
-   SeparateInstances {{ collectd_settings.plugins.write_graphite.separateinstances|lower }}
-   StoreRates {{ collectd_settings.plugins.write_graphite.storerates|lower }}
-   AlwaysAppendDS {{ collectd_settings.plugins.write_graphite.alwaysappendds|lower }}
- </Carbon>
+{%- for node_name, graphite_node in collectd_settings.plugins.write_graphite.nodes.iteritems() %}
+ <Node "{{ node_name }}">
```

Comment:
This hurts my pride to admit, but I can't seem to figure out how to go about this. I could have sworn I saw something similar in the Salt docs, but my google-fu is failing me. Any chance you have a project that implements such a pattern that I could reference? Or the a place in the docs that might be of relevance.